Libs/Common/Shapeworks.cpp

```diff
@@ -13,9 +13,13 @@ Coord toCoord(const Dims &d) {
                 static_cast<itk::IndexValueType>(d[1]),
                 static_cast<itk::IndexValueType>(d[2])}); }
 Dims toDims(const Coord &c) {
-  return Dims({static_cast<itk::SizeValueType>(c[0]),
-               static_cast<itk::SizeValueType>(c[1]),
-               static_cast<itk::SizeValueType>(c[2])}); }
+  return Dims({static_cast<Dims::value_type>(c[0]),
```

Comment:
Dims is itk::Size, which is unsigned, so let's make sure these are std::max(0,arg) when converted from both Coord and Point.

Reply:
fixed